charts/rancher-eks-operator/1.0.600/Chart.yaml

```diff
@@ -4,5 +4,5 @@ description: A Helm chart for provisioning EKS clusters
 home: https://github.com/rancher/eks-operator
 sources:
   - "https://github.com/rancher/eks-operator"
-version: 1.0.500
-appVersion: v1.0.5
+version: 1.0.600
+appVersion: v1.0.6-rc1
```

Comment:
has 1.0.500 already been in a release?

Reply:
Yes, I believe it was in 2.5.4 for private endpoint stuff.

Reply:
This is easy to check by looking at release branch, it was added: https://github.com/rancher/system-charts/tree/release-v2.5/charts/rancher-eks-operator